modules/migration/file_format.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migration
+
+import (
+	"encoding/json"
```

Comment:
Sigh, we really need to get a linter for this specific code.

`encoding/json` should be replaced by `code.gitea.io/gitea/modules/json`

Reply:
Fixed.